docs/resources/ga_health_check.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Global Accelerator (GA)"
+---
+
+# huaweicloud_ga_health_check
+
+Manages a GA health check resource within HuaweiCloud.
+
+## Example Usage
+
+```HCL
+variable "endpoint_group_id" {}
+
+resource "huaweicloud_ga_health_check" "test" {
+  endpoint_group_id = var.endpoint_group_id
+  enabled           = true
+  interval          = 10
+  max_retries       = 5
+  port              = 8001
+  timeout           = 10
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `endpoint_group_id` - (Required, String, ForceNew) Specifies the endpoint group ID.
+
+  Changing this parameter will create a new resource.
+
+* `enabled` - (Required, Bool) Specifies whether to enable health check.
+
+* `interval` - (Required, Int) Specifies the health check interval, in seconds.
+
+* `max_retries` - (Required, Int) Specifies the maximum number of retries.
+  Specifies the number of consecutive health checks when the health check result of an endpoint changes
```

Comment:
it's hard to understand because users can not get the check result from terraform.

Is it possible to add an attribute of the health check result?

Reply:
there is no API response about this, so there is no way to add an attribute of the health check result